### ISSUE TITLE
CMake fails to detect missing `sparsehash` dependency

### ISSUE THREAD

**Issue by cipriancraciun** [mhx/dwarfs#2]
I'm trying to build `0.2.0` on OpenSUSE Tumbleweed, and after I've successfully run `cmake` and started building it broke stating it can't find `sparsehash` dependency:
~~~~
[ 92%] Linking CXX static library libfolly.a
/tmp/dwarfs-0.2.0/src/dwarfs/block_manager.cpp:32:10: fatal error: sparsehash/dense_hash_map: No such file or directory
   32 | #include <sparsehash/dense_hash_map>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/dwarfs.dir/build.make:108: CMakeFiles/dwarfs.dir/src/dwarfs/block_manager.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 92%] Built target folly
make[1]: *** [CMakeFiles/Makefile2:284: CMakeFiles/dwarfs.dir/all] Error 2
make: *** [Makefile:171: all] Error 2
~~~~

I think and extra check in CMake should solve this.

**Comment by mhx** [comment]
Agreed, thanks for the report.

**Comment by mhx** [comment]
Should be fixed by cd42e21e2b68187c3dc8971a1b46edf412c8351a. (Please re-open if there are still issues.)
